CMakeLists.txt

```diff
@@ -19,9 +19,9 @@ set(base_files
   src/gennodejs/base_deserialize.js
   src/gennodejs/find.js)
 
-file(COPY ${base_files} DESTINATION ${CATKIN_DEVEL_PREFIX}/share/node_js)
+file(COPY ${base_files} DESTINATION ${CATKIN_DEVEL_PREFIX}/share/gennodejs)
 install(FILES ${base_files}
-  DESTINATION ${CATKIN_GLOBAL_SHARE_DESTINATION}/node_js)
+  DESTINATION ${CATKIN_GLOBAL_SHARE_DESTINATION}/gennodejs)
```

Comment:
Ah, well that does make sense. Alrighty, it's not a big deal, but the `file(COPY ...)` should go.